community/kernel/src/main/java/org/neo4j/kernel/impl/core/NodeProxy.java

```diff
@@ -377,6 +382,85 @@ public Object getProperty( String key, Object defaultValue )
     }
 
     @Override
+    public Map<String, Object> getProperties( String... keys )
```

Comment:
We can add a check `keys == null || keys.length == 0` and do a quick return of `Collections.emptyMap()`